mpf/devices/ball_routing.py

```diff
@@ -26,9 +27,14 @@ def __init__(self, machine: "MachineController", name: str) -> None:
         self._balls_at_target = 0
         self._handler = []
 
-    def enable(self, **kwargs):
-        """Enable routing."""
+    @event_handler(2)
+    def event_enable(self, **kwargs):
```

Comment:
For `event_enable` handlers like this, would the default behavior inherited from `ModeDevice` suffice? Or is there a reason to explicitly define it here?

Reply:
Looks like I oversaw that. Will fix it.

Reply:
done